Dockerfile.rootless

```diff
@@ -56,7 +56,7 @@ COPY docker/rootless /
 COPY --from=build-env --chown=root:root /go/src/code.gitea.io/gitea/gitea /usr/local/bin/gitea
 COPY --from=build-env --chown=root:root /go/src/code.gitea.io/gitea/environment-to-ini /usr/local/bin/environment-to-ini
 
-USER git:git
+USER 1000:1000
```

Comment:
```suggestion
USER 1000:1000 # git:git
```

Reply:
Hi @ibotty. Never faced this issue with both Gitea images. 🤔

(Out of interest: Are you referring to the deprecated `PodSecurityPolicy` or the pod-own `securityContext` object?)

In any case you can always define how the pod will run. This overrides the image specification in some kind.

```yaml
securityContext:
  runAsUser: 1000
  runAsNonRoot: true
```

Modifying the Dockerfiles wouldn't be necessary, I think.

Reply:
Well, this happens on OpenShift/OKD.  I know I can work around with using `securityContext` and I am doing that.  But e.g. using the helm chart will not easily allow specifying it.  And I don't see any harm in doing that because the IDs are explicitly set earlier.  I can, of course, incude a test that checks whether the uid of the git user matches 1000.

Reply:
I'm ok with it as you mentioned id's are explicite set